src/index.html

```diff
@@ -1,63 +1,85 @@
-    />
+    /-->
     <link rel="stylesheet" type="text/css" href="./css/style.css" />
     <link rel="stylesheet" type="text/css" href="./css/header.css" />
 
     <link rel="shortcut icon" href="./assets/favicon.ico" />
-</head>
-<body>
+  </head>
+  <body>
     <header>
-        <div class="header-container">
-          <div class="left-header">
-            <img
-              src="./assets/daily.svg"
-              class="daily-logo"
-              alt="Daily logo reads Daily"
-            />
-            <strong>
-              <p class="header-title">Daily demo</p>
-            </strong>
-          </div>
-          <div class="right-header">
-            <a
-              href="https://docs.daily.co/reference"
-              target="_blank"
-              class="link"
-            >
-              <button class="button docs-button white-button">
-                API docs
-                <img
-                  class="arrow-icon"
-                  src="./assets/new-tab-icon.png"
-                  alt="Arrow icon points up and to the right"
-                  aria-hidden="true"
-                />
-              </button>
-            </a>
-            <span class="divider"></span>
-            <a href="https://github.com/daily-demos/boilerplate-vanilla-js/" target="_blank">
+      <div class="header-container">
+        <div class="left-header">
+          <img
+            src="./assets/daily.svg"
+            class="daily-logo"
+            alt="Daily logo reads Daily"
+          />
+          <strong>
+            <p class="header-title">Daily Prejoin Presence Demo</p>
+          </strong>
+        </div>
+        <div class="right-header">
+          <a
+            href="https://docs.daily.co/reference"
+            target="_blank"
+            class="link"
+          >
+            <button class="button docs-button white-button">
+              API docs
               <img
-                class="octocat"
-                src="./assets/github.png"
-                alt="GitHub logo Octocat"
+                class="arrow-icon"
+                src="./assets/new-tab-icon.png"
+                alt="Arrow icon points up and to the right"
+                aria-hidden="true"
               />
-            </a>
+            </button>
+          </a>
+          <span class="divider"></span>
+          <a
+            href="https://github.com/daily-demos/boilerplate-vanilla-js/"
+            target="_blank"
+          >
+            <img
+              class="octocat"
+              src="./assets/github.png"
+              alt="GitHub logo Octocat"
+            />
+          </a>
+        </div>
+      </div>
+    </header>
+    <noscript>
+      This application requires JavaScript to run. Please enable JavaScript in
+      your browser.
+    </noscript>
+    <div id="container">
+      <div id="entry">

```

Comment:
I think it would be good to add a header and short description (h1 + p) in here to give a little more information for the entry view. Even though they've _probably_ just read the README, you should be able to look at the UI without prior knowledge and know what you're about to interact with.
So, something like: "Daily Prejoin Presence Demo", "Create a new private room and join the lobby to see who else is already in the call."

Aside: I was just looking to see what this button does on click (e.g. is it a private room?) and the actual Daily stuff is pretty abstracted away. I would add more info to the README that you've written it in Go and where to look to see the Daily-specific sections. It's a bit confusing if you (me!) aren't familiar with Go. And I think it's the first demo with Go so it might be unexpected for someone expecting JS (also me hah!) It's a nice addition but I think it's good to give all that upfront in the README :)

Reply:
Ty, added some more info to the entry page and README